docs/Extension/guides/backup-and-restore.md

```diff
@@ -0,0 +1,109 @@
+The Backup and Restore feature of the Universal Profiles Browser Extension allows users to export all of their UPs to a password-protected file to be restored later.
+
+:::info Available Recovery Options
+
+Other safe options to restore your Universal Profile:
+
+- Adding an email or 2FA recovery feature via my.universalprofile.cloud
+- Setting another wallet as a recovery controller, known as third-party wallet recovery
+
+:::
+
+## Accessing Backup and Restore
+
+Navigate to the extension settings and locate the Backup and Restore section.
+
+<div style={{display: 'flex', gap: '10px', justifyContent: 'center', alignItems: 'flex-start', flexWrap: 'wrap'}}>
+  <img src="/img/extension/backup-and-restore/up-settings.png" alt="Extension Settings" style={{maxHeight: '500px', width: 'auto', maxWidth: '45%', minWidth: '280px', objectFit: 'contain'}} />
+  <img src="/img/extension/backup-and-restore/bnr section.png" alt="Backup and Restore Section" style={{maxHeight: '500px', width: 'auto', maxWidth: '45%', minWidth: '280px', objectFit: 'contain'}} />
+</div>
+
+## 1. Backup Profiles
+
+This feature lets you back up all Universal Profiles (UPs) currently loaded in your browser extension, for both LUKSO Mainnet and Testnet.
+
+<div style={{display: 'flex', justifyContent: 'center'}}>
+  <img src="/img/extension/backup-and-restore/backup-profiles.png" alt="Select Profiles" style={{maxHeight: '500px', width: 'auto'}} />
+</div>
+
+When backing up your Universal Profiles, you have two options:
+
+- **Backup with password protection:** This encrypts your backup file and adds an extra security for your UPs.
+- **Backup without password protection:** This option is less secure, as your backup file will not be encrypted.
+
+### a. Backup with Protection (Recommended)
+
+Choose to back up with password protection for enhanced security.
+
+<div style={{display: 'flex', gap: '10px', justifyContent: 'center', alignItems: 'flex-start', flexWrap: 'wrap'}}>
+  <img src="/img/extension/backup-and-restore/backup-w-password.png" alt="Backup with Password Option" style={{maxHeight: '500px', width: 'auto', maxWidth: '45%', minWidth: '280px', objectFit: 'contain'}} />
+  <img src="/img/extension/backup-and-restore/backup-w-password-form.png" alt="Enter Password" style={{maxHeight: '500px', width: 'auto', maxWidth: '45%', minWidth: '280px', objectFit: 'contain'}} />
+</div>
+
+Enter a strong password and a hint to help you remember this password.
+
+### b. Backup without Protection
+
+Alternatively, you can create a backup without password protection (not recommended for security reasons).
+
+<div style={{display: 'flex', justifyContent: 'center', paddingBottom: '24px'}}>
+  <img src="/img/extension/backup-and-restore/backup-wout-password.png" alt="Backup without Password" style={{maxHeight: '500px', width: 'auto'}} />
+</div>
+
+:::warning Storing the file
+
+Save your backup file securely. Make sure to remember your password as it will be required for restoring your UPs.
```

Comment:
Ah you mentioned this here I see. Suggest moving it just under the heading of option a (with password)